### DISCUSSION TITLE
System Tray

### DISCUSSION THREAD

**Discussion by BolingbrookIL** [Open-Shell/Open-Shell-Menu#1308] [Q&A]
At the bottom right corner of my screen, my system tray containing the date/time, speaker volume, internet access, etc. appears BEHIND the black colored banner. I can see the icons if I hover over them, but they are not otherwise visible. Did I incorrectly set this via Open-Shell? Would appreciate feedback on how to restore visibility. Thank you in advance for your feedback.

![image](https://user-images.githubusercontent.com/123646551/214858458-44da5e7b-0c6d-40fb-aa3e-8b0820b4f392.png)

![image](https://user-images.githubusercontent.com/123646551/214858557-7a65b144-c8e6-4b7b-9d7e-ccaa2005b59b.png)

.

**Comment by ge0rdi** [comment]
Do you have `Customize taskbar` setting enabled in `Open-Shell` settings (`Taskbar` tab)?

**Reply by BolingbrookIL** [reply]
Yes.
![image](https://user-images.githubusercontent.com/123646551/214865682-aa082ee0-56f4-47ce-bfb8-af057ad2b80b.png)

**Reply by ge0rdi** [reply]
What is the version of `Open-Shell` and Windows you are using?

**Reply by BolingbrookIL** [reply]
Thank you for helping. I'm using Open Shell Version 4.4.170. I'm running Windows 11.

**Reply by ge0rdi** [reply]
Please, use latest [pre-release](https://github.com/Open-Shell/Open-Shell-Menu/releases) on Windows 11.

What exact Windows 11 version do you have? 21H2 / 22H2 / Insider ?
For example `Glass` option doesn't work since 22H2.
And taskbar coloring doesn't work in insider builds at all.

**Reply by BolingbrookIL** [reply]
Here is the information you requested

![image](https://user-images.githubusercontent.com/123646551/214932571-4a4b9e18-5cbc-4f84-b20b-2edf33142247.png)

**Comment by BolingbrookIL** [comment]
It appears my current version is 22H2. Should I use the download link you sent earlier? And by the way, thank you for responding to my question. I appreciate it!

**Reply by ge0rdi** [reply]
> Should I use the download link you sent earlier? 

Yes. Windows 11 is not supported by current official release.

**Comment by BolingbrookIL** [comment]
I don’t know who you are, but I totally appreciate your advice. I’ll install the version you suggest and will let you know of the outcome.

**Reply by BolingbrookIL** [reply]
Quick question……do I need to eliminate the 2 “X’s” below and download everything??

![image](https://user-images.githubusercontent.com/123646551/215125360-15c01fd7-f0cb-4c4f-a161-026e96226be7.png)

**Reply by ge0rdi** [reply]
No.

**Comment by bonzibudd** [comment]
Another note; If you are using a darker color for the Taskbar background, you might want to set the Windows color to Dark so that the text is more visible. 
![image](https://user-images.githubusercontent.com/61938331/215150781-b3277ea7-8703-4aec-8c89-14bd3cd2f82f.png)

This won't affect app background colors.

**Comment by BolingbrookIL** [comment]
Thank you, Eli…..much appreciated!
